src/eduid_common/api/utils.py

```diff
@@ -80,7 +101,7 @@ def save_dashboard_user(user):
     if isinstance(user, User) and not isinstance(user, DashboardUser):
         # turn it into a DashboardUser before saving it in the dashboard private db
         user = DashboardUser(data = user.to_dict())
-    current_app.dashboard_userdb.save(user)
+    getattr(current_app, dbattr_name).save(user)
```

Comment:
I think we need to be very clear and concise about databases. Either this function should only save to the database loaded as current_app.dashboard_userdb or take mandatory argument with the database the DashboardUser object should be saved to,